set.mm

```diff
@@ -672232,6 +672338,136 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
+       ` N e. NN0 ` .  (Contributed by SN, 22-Mar-2025.) $)
+    fz1sump1 $p |- ( ph -> sum_ k e. ( 1 ... ( N + 1 ) ) A =
+                             ( sum_ k e. ( 1 ... N ) A + B ) ) $=
+      ( c1 caddc co cfz csu cmin cn cuz cfv cn0 wcel nn0p1nn nnuz fsumm1 nn0cnd
+      syl eleqtrdi 1cnd pncand oveq2d sumeq1d oveq1d eqtrd ) AIEIJKZLKBDMIULINK
+      ZLKZBDMZCJKIELKZBDMZCJKABCDIULAULOIPQAERSULOSFETUDUAUEGHUBAUOUQCJAUNUPBDA
+      UMEILAEIAEFUCAUFUGUHUIUJUK $.
+  $}
+
+  ${
+    $d N k $.
+    $( The Odd Number Theorem.  The sum of the first ` N ` odd numbers is
+       ` N ^ 2 ` .  A corollary of ~ arisum .  (Contributed by SN,
+       21-Mar-2025.) $)
+    oddnumth $p |-
+      ( N e. NN0 -> sum_ k e. ( 1 ... N ) ( ( 2 x. k ) - 1 ) = ( N ^ 2 ) ) $=
+      ( cn0 wcel c1 cfz co c2 cmul cmin csu cexp caddc fzfid 2cnd elfznn adantl
+      cv cc 1cnd nncnd mulcld fsumsub cdiv arisum oveq2d fsummulc2 nn0cn addcld
+      wa sqcld cc0 wne 2ne0 a1i divcan2d 3eqtr3d id fz1sumconst mulridd oveq12d
+      eqtrd pncand 3eqtrd ) BCDZEBFGZHARZIGZEJGAKVFVHAKZVFEAKZJGBHLGZBMGZBJGVKV
+      EVFVHEAVEEBNZVGVFDZVHSDVEVNHVGVNOVNVGVGBPUAZUBQVEVNUJTUCVEVIVLVJBJVEHVFVG
+      AKZIGHVLHUDGZIGVIVLVEVPVQHIABUEUFVEVFVGHAVMVEOZVNVGSDVEVOQUGVEVLHVEVKBVEB
+      BUHZUKZVSUIVRHULUMVEUNUOUPUQVEVJBEIGBVEEABVEURVETUSVEBVSUTVBVAVEVKBVTVSVC
+      VD $.
+
+    $( Nicomachus's Theorem.  The sum of the odd numbers from ` N ^ 2 - N + 1 `
+       to ` N ^ 2 + N - 1 ` is ` N ^ 3 ` .  Proof 2 from
+       ~ https://proofwiki.org/wiki/Nicomachus%27s_Theorem .  (Contributed by
+       SN, 21-Mar-2025.) $)
+    nicomachus $p |- ( N e. NN0 ->
+      sum_ k e. ( 1 ... N ) ( ( ( N ^ 2 ) - N ) + ( ( 2 x. k ) - 1 ) )
+                                                               = ( N ^ 3 ) ) $=
+      ( cn0 wcel c1 cfz co c2 cexp cmin cmul caddc csu c3 cc subcld a1i oveq12d
+      sqcld 3eqtrd cv fzfid nn0cn adantr 2cnd elfznn adantl mulcld 1cnd fsumadd
+      wa nncnd fz1sumconst subdid df-3 oveq2i 2nn0 expp1d eqtrid mulcomd eqtr2d
+      id sqvald eqcomd oddnumth 3nn0 expcld npcand ) BCDZEBFGZBHIGZBJGZHAUAZKGZ
+      EJGZLGAMVJVLAMZVJVOAMZLGBNIGZVKJGZVKLGVRVIVJVLVOAVIEBUBVIVMVJDZUKZVKBWABV
+      IBODVTBUCZUDZSWCPWAVNEWAHVMWAUEVTVMODVIVTVMVMBUFULUGUHWAUIPUJVIVPVSVQVKLV
+      IVPBVLKGBVKKGZBBKGZJGVSVIVLABVIVBVIVKBVIBWBSZWBPUMVIBVKBWBWFWBUNVIWDVRWEV
+      KJVIVRVKBKGZWDVIVRBHELGZIGWGNWHBIUOUPVIBHWBHCDVIUQQURUSVIVKBWFWBUTVAVIVKW
+      EVIBWBVCVDRTABVERVIVRVKVIBNWBNCDVIVFQVGWFVHT $.
+
+    $d N k l m x y $.
+    $( The sum of the first ` N ` perfect cubes is the sum of the first ` N `
+       nonnegative integers, squared.  This is the Proof by Nicomachus from
+       ~ https://proofwiki.org/wiki/Sum_of_Sequence_of_Cubes using induction
+       and index shifting to collect all the odd numbers.  (Contributed by SN,
+       22-Mar-2025.) $)
+    sumcubes $p |- ( N e. NN0 -> sum_ k e. ( 1 ... N ) ( k ^ 3 )
+                             = ( sum_ k e. ( 1 ... N ) k ^ 2 ) ) $=
```

Comment:
It would be cool to do a numeric application for N=9... without waiting for next year!  :)